src/clicache/src/CacheableDate.cpp

```diff
@@ -90,7 +90,7 @@ namespace Apache
           TimeSpan epochSpan = m_dateTime - EpochTime;
           int64_t millitime =
             epochSpan.Ticks / TimeSpan::TicksPerMillisecond;
-          m_hashcode = (int)millitime ^ (int)((int64)millitime >> 32);
+          m_hashcode = (int)millitime ^ (int)((int64_t)millitime >> 32);
```

Comment:
This version of CacheableDate is C++/CLI. C++/CLI is only C++ in name. It does not make sense to use C++11 types in C++/CLI despite the fact that they "work". In C++/CLI `int`, `long`, etc. are not ambiguous like they are in C++.